modules/search-service-impl/src/test/java/org/opencastproject/search/impl/persistence/SearchServicePersistenceTest.java

```diff
@@ -162,7 +162,15 @@ public void testRetrieving() throws Exception {
 
     Date deletionDate = new Date();
     searchDatabase.deleteMediaPackage(mediaPackage.getIdentifier().toString(), deletionDate);
-    episode = searchDatabase.getMediaPackage(mediaPackage.getIdentifier().toString());
```

Comment:
This now throws a NotFoundException. Interestingly, the episode variable wasn't used here.